src/analysis/cohort.py

```
@@ -230,6 +236,67 @@ def extract_milestones(
     return df
 
 
+def build_milestone_table(
+    milestone_df: pd.DataFrame,
+    milestones: Sequence[int] = RETENTION_MILESTONES,
+) -> pd.DataFrame:
+    """코호트 × 마일스톤(M1/M3/M6/M12) 와이드 수치표를 만든다.
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)